### PR TITLE
Repathed all Box station floors that had var edited icon states. Also generic z levels

### PR DESCRIPTION
**Pull request by xxalpha**
[[ Need to do one last thing before this is merge ready ]]
--------------------------

Besides repathing floors to existing types, I also made some new airless types:

 - /turf/open/floor/plasteel/airless/black
 - /turf/open/floor/plasteel/airless/white
 - /turf/open/floor/plasteel/airless/warning
 - /turf/open/floor/plasteel/airless/warning/corner
 - /turf/open/floor/plasteel/airless/warnplate
 - /turf/open/floor/plasteel/airless/circuit
 - /turf/open/floor/plasteel/airless/circuit/gcircuit
 - /turf/open/floor/plasteel/airless/asteroid
 - /turf/open/floor/plasteel/airless/asteroid/warning
 - /turf/open/floor/plasteel/airless/floorgrime
 - /turf/open/floor/plasteel/airless/solarpanel
 - /turf/open/floor/engine/airless

Also all maps that weren't TGM already have been converted.

For this PR, only Box station map and the generic z2, z3, z4, z5, z6 maps have been edited.

http://pastebin.com/raw/ZtkeVGen

Related issue: https://github.com/tgstation/tgstation/issues/17946

### CODE REVIEW

**Comment by Bawhoppen** [comment]
i think you fucked up?

**Comment by xxalpha** [comment]
Do you realize who you are talking to?

**Comment by pudl** [comment]
Please dont make it a standard to use pathed floors, icon editing is so much easier for mappers and this standard really doesnt mean anything

**Comment by xxalpha** [comment]
Too bad you are wrong.

**Comment by Bawhoppen** [comment]
>make one post

>xxalpha goes feral in the coderbus yelling I'm a shitposting autist

File: _maps/map_files/generic/z2.dmm [hunk 1]
```diff
@@ -94,28 +94,24 @@
 /turf/open/floor/holofloor/plating,
 /area/holodeck/rec_center/offline)
 "as" = (
-/turf/open/floor/holofloor{
```
Comment (ChangelingRain) [inline _maps/map_files/generic/z2.dmm]: Uh, you might not want to replace these

Comment (xxalpha) [inline _maps/map_files/generic/z2.dmm]: Yeah I noticed it just now.

Comment (ChangelingRain) [inline _maps/map_files/generic/z2.dmm]: As in, otherwise people can use the holodeck to farm floor materials 'might not want to'

**Comment by RandomMarine** [comment]
After the freeze somebody should make floorgrime tiles cleanable.

**Comment by Aranclanos** [comment]
holy shit you fucking autists actually finished it  :heart: :heart: :heart:

**Comment by xxalpha** [comment]
This is ready to be merged. @Cheridan

**Comment by xxalpha** [comment]
Read the OP next time, oranges.

**Pull request merged**